common/changes/@microsoft/rush/user-danade-MorePackageExtractorPatterns_2023-08-01-09-05.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Add \"patternsToInclude\" and \"patternsToExclude\" support to Rush deploy.json configurations. This will allow developers to include or exclude provided glob patterns within a local project when running \"rush deploy\"",
```

Comment:
```suggestion
      "comment": "Add `patternsToInclude` and `patternsToExclude` support to Rush deploy.json configurations. This will allow developers to include or exclude provided glob patterns within a local project when running `rush deploy`.",
```